swarm/pss/api.go

```diff
@@ -92,7 +92,7 @@ func (pssapi *API) Receive(ctx context.Context, topic Topic, raw bool, prox bool
 }
 
 func (pssapi *API) GetAddress(topic Topic, asymmetric bool, key string) (PssAddress, error) {
-	var addr *PssAddress
+	var addr PssAddress
```

Comment:
Why are there changes related to PSS in this PR? Shouldn't that be in a separate one? I have been instructed so for other PRs